plugins/woocommerce/src/Internal/Admin/FeaturePlugin.php

```diff
@@ -115,7 +120,7 @@ protected function define_constants() {
 		 * @deprecated 3.3.0
 		 * @var string
 		 */
-		define( 'WC_ADMIN_VERSION_NUMBER', '3.3.0' );
+		$this->define( 'WC_ADMIN_VERSION_NUMBER', '3.3.0' );
```

Comment:
This was throwing an admin notice since `WC_ADMIN_VERSION_NUMBER` gets defined twice. It prevents plugin activation in some cases since it causes unexpected output.

<img width="875" alt="image" src="https://user-images.githubusercontent.com/3747241/165692484-9a19ce5f-b1ba-4f13-9209-44a32c2ec2ce.png">

Reply:
```suggestion
		if ( ! defined( 'WC_ADMIN_VERSION_NUMBER' ) ) {
			/**
			  * Define the current WC Admin version.
			  *
 			  * @deprecated 3.3.0
 			  * @var string
 			  */
			define( 'WC_ADMIN_VERSION_NUMBER', '3.3.0' );
		}
```

How about using `defined` to make this check rather than the `$this->define` helper? 

While it is longer, it would allow `WC_ADMIN_VERSION_NUMBER` to be deprecated explicitly with the `@deprecated` tag, and IDEs like PHPStorm can use that info to warn devs about any future usage.

Reply:
@adrianduffell That's a great suggestion, I wasn't aware that this change would've affected the deprecation tag. Thanks!